services/npms-io/score.service.js

```diff
@@ -0,0 +1,85 @@
+'use strict'
+
+const Joi = require('joi')
+const { BaseJsonService } = require('..')
+const { coveragePercentage } = require('../color-formatters')
+
+// https://api-docs.npms.io/#api-Package-GetPackageInfo
+const numberSchema = Joi.number().min(0).max(1).required()
+const responseSchema = Joi.object({
+  score: Joi.object({
+    final: numberSchema,
+    detail: Joi.object({
+      maintenance: numberSchema,
+      popularity: numberSchema,
+      quality: numberSchema,
+    }),
+  }),
+}).required()
+
+const keywords = ['node', 'npm score']
+
+module.exports = class NpmsIO extends BaseJsonService {
+  static category = 'analysis'
+
+  static route = {
+    base: 'npms-io',
+    pattern:
+      ':type(final|maintenance|popularity|quality)/:scope(@.+)?/:packageName',
```

Comment:
I know we have a couple existing badge routes that are just `/serviceName/quality/:params` but I still think we'd be better served by having more explicit nouns here, e.g.:

```suggestion
      ':type(final|maintenance|popularity|quality)-score/:scope(@.+)?/:packageName',
```

This is partly because some of the types look/sound strange (particularly `npms-io/final/...`, and also because there are other non-score data points under those exact same terms (maintenance, popularity, quality) and we could potentially need to provide badges for in the future (we have open requests for at least one already).

Note the test routes would need to be updated accordingly as well